<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestMe2PyModuleWidget</class>
 <widget class="qSlicerWidget" name="TestMe2PyModuleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>365</width>
    <height>319</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TestMe2</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="ctkCollapsibleButton" name="CTKCollapsibleButton">
     <property name="text">
      <string>Parameters</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <layout class="QGridLayout" name="gridLayout">
        <item row="2" column="0">
         <widget class="QLabel" name="LabelPosition">
          <property name="text">
           <string>Height</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="LabelTransform">
          <property name="text">
           <string>Transform</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="LabelFiducial">
          <property name="text">
           <string>Fiducial</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="qMRMLNodeComboBox" name="InputFiducial">
          <property name="nodeTypes">
           <stringlist>
            <string>vtkMRMLMarkupsFiducialNode</string>
           </stringlist>
          </property>
          <property name="hideChildNodeTypes">
           <stringlist notr="true"/>
          </property>
          <property name="interactionNodeSingletonTag">
           <string notr="true"/>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="qMRMLNodeComboBox" name="InputTransform">
          <property name="nodeTypes">
           <stringlist>
            <string>vtkMRMLLinearTransformNode</string>
           </stringlist>
          </property>
          <property name="hideChildNodeTypes">
           <stringlist/>
          </property>
          <property name="interactionNodeSingletonTag">
           <string notr="true"/>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="ctkSliderWidget" name="HeightSlider">
          <property name="minimum">
           <double>-100.000000000000000</double>
          </property>
          <property name="maximum">
           <double>100.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="CreateButton">
        <property name="text">
         <string>Create Markup Point</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkSliderWidget</class>
   <extends>QWidget</extends>
   <header>ctkSliderWidget.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>TestMe2PyModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>InputTransform</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>182</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>TestMe2PyModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>InputFiducial</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>182</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>214</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
